integration-tests/http/__tests__/claims/claims.spec.ts

```diff
@@ -742,7 +742,7 @@ medusaIntegrationTestRunner({
           expect(paymentCollections[0]).toEqual(
```

Comment:
question: Could I get you to add a few sentences explaining why the totals changed here? I presume it is due to the changes, but it would be good to understand exactly which of them corrected this